front/src/containers/Authentication/styleLogin.js

```diff
@@ -0,0 +1,66 @@
+import styled from 'styled-components'
+
+export const Container = styled.SafeAreaView`
+  flex: 1;
+  background: #333;
+  justify-content: center;
+  align-items: center;
+`
+export const ContainerView = styled.View `
```

Comment:
flex 1 don't is needed, you can use a View component of react native to group the login inputs, correct?